### ISSUE TITLE
Blocked by parental control / 162.220.223.23

### ISSUE THREAD

**Issue by bcookatpcsd** [AdguardTeam/AdGuardHome#5679]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not ask a question

### Operating system type

Linux, Other (please mention the version in the description)

### CPU architecture

AMD64

### Installation

Docker

### Setup

On one machine

### AdGuard Home version

107.26

### Description

All of these are coming back as parental control blocked for some reason.. 

![image](https://user-images.githubusercontent.com/55087301/229583096-99af77ae-1440-40ff-b91b-daf20777b8a6.png)

EST-4 here..

**Comment by ainar-g** [comment]
I've tried a couple of domains from this list, and most are either explicitly in the parental control list, or their IPs/CNAMEs are.

If you think that those are errors, please report using the [new issue form](https://reports.adguard.com/en/new_issue.html), as AdGuard Home merely uses the provided data.
